plugins/source/homebrew/main.go

```diff
@@ -1,12 +1,15 @@
 package main
 
 import (
+	"context"
+
 	"github.com/cloudquery/cloudquery/plugins/source/homebrew/resources/plugin"
-	"github.com/cloudquery/plugin-sdk/v3/serve"
+	"github.com/cloudquery/plugin-sdk/v4/serve"
 )
 
 const sentryDSN = "https://e944e0a50f1c43eaa6ffd89e7b39f4da@o1396617.ingest.sentry.io/4504734217142272"
 
 func main() {
-	serve.Source(plugin.Plugin(), serve.WithSourceSentryDSN(sentryDSN))
+	p := serve.Plugin(plugin.Plugin(), serve.WithPluginSentryDSN(sentryDSN))
+	p.Serve(context.Background())
```

Comment:
We seem to be ignoring `error` result from Serve, which gets caught by the linter.